### PR TITLE
DOCSP-22673 Create FAQ page

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22673-Create-faq-page/faq/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-22673)

### CODE REVIEW

File: source/faq.txt [hunk 1]
```diff
@@ -10,3 +10,71 @@ Frequently Asked Questions
...
+<mongodb-uri>` to connect to the source and destination clusters. 
+
+:ref:`LDAP <security-auth-ldap>` and :ref:`X509
+<authentication-mechanism-x509>` are supported. For available
+authentication options, see :ref:`authentication`.
+
+
+Can I configure ``mongosync`` for high availability?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```
Comment (jmd-mongo) [inline source/faq.txt:60]: [suggestion]

Consider moving the paragraph that begins "There is no automatic failover ..." to the top as I think this more clearly addresses the question. I had to read this a few times before I could figure out exactly what the answer actually was as currently written.

File: source/faq.txt [hunk 2]
```diff
@@ -10,3 +10,71 @@ Frequently Asked Questions
...
+There is no automatic failover built into ``mongosync``, however you
+can write a script or use your operating system's process managers,
+``systemd`` for example, to restart the ``mongosync`` process. 
+
+
+Can the source or destination be a replica set with arbiters? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. The source replica-set must have more than 2 non-arbiter node.
```
Comment (jmd-mongo) [inline source/faq.txt:77]: [minor typos]

I think we want "replica set" and "nodes" on line 77.

File: source/faq.txt [hunk 3]
```diff
@@ -10,3 +10,71 @@ Frequently Asked Questions
...
+Please refer to read preference options to specify in the source cluster’s
```
Comment (jmd-mongo) [inline source/faq.txt:78]: [comment]

I am having a hard time understanding what "refer to read preference options to specify in the source cluster's connection string" means. I'd consider rewriting.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @davemungo. I've left a few suggestions, questions, and a comment.

Thanks!
Joe

File: source/faq.txt [hunk 4]
```diff
@@ -10,3 +10,71 @@ Frequently Asked Questions
 This page provides answers to some frequently asked questions we have
 encountered. If you have additional questions please contact MongoDB
 Support.
+
+Does ``mongosync`` have to use the same operating system as MongoDB Enterprise Server? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When ``mongosync`` runs on its own hardware, it does not have to use
```
[the remark this answers is not in the document]
Comment (jmd-mongo) [reply source/faq.txt:17]: Got it! Thanks for clarifying.

File: source/faq.txt [hunk 5]
```diff
@@ -10,3 +10,82 @@ Frequently Asked Questions
...
+
+:ref:`LDAP <security-auth-ldap>` and :ref:`X509
+<authentication-mechanism-x509>` are supported. For available
+authentication options, see :ref:`authentication`.
+
+
+Can I configure ``mongosync`` for high availability?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is no automatic failover built into ``mongosync``, however you
```
Comment (jmd-mongo) [inline source/faq.txt:70]: [suggestion]

Consider splitting this into two sentences: "There is no automatic failover built into ``mongosync``. However, ..."

Comment (davemungo) [reply source/faq.txt]: Done

**Review by davemungo — COMMENTED** [review]
Thanks @jmd-mongo sending out for tech review next

**Comment by davemungo** [comment]
@alan-zheng-mdb  The FAQ is ready for review when you have a moment.

**Review by alan-zheng-mdb — APPROVED** [review]
LGTM

**Pull request merged**